src/api/lessons.ts

```diff
@@ -0,0 +1,16 @@
+import axios, { AxiosResponse } from 'axios';
+import buildUrl from './buildUrl';
+
+export async function getLessonById(lessonId: number): Promise<AxiosResponse> {
+  try {
+    return await axios.get(buildUrl(`/lessons/${lessonId}`));
+  } catch (err) {
+    return err;
+  }
+}
+
+const lessons = {
+  getLessonById,
+}
+
+export default lessons;
```

Comment:
Is this necessary if we're exporting the function anyways? Not 100% sure how it works, so might be wrong tbh

Reply:
Technically I should get rid of the `export function` at the top, so we only export the `lessons`.